### DISCUSSION TITLE
Welcome to discord.py Discussions!

### DISCUSSION THREAD

**Discussion by Rapptz** [Rapptz/discord.py#5831] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

To get started, comment below with an introduction of yourself and tell us about what you do with this community.

Time to try out this feature. I plan on converting a lot of the question tagged issues to this format for now.

**Comment by ioistired** [comment]
Hi there! I'm io mintz. You may know me as lambda#0987 on the Discord. I've been a member of this community since the DAPI days, and have made a lot of friends and bots since then. I enjoy writing (templated) SQL and thinking about open-source licenses. You can find my bots on my GitHub profile.

**Comment by scrazzz** [comment]
i make bot.
